### ISSUE TITLE
Toolbox Update Leap Seconds - Failure to Update

### ISSUE THREAD

**Issue by Caseyb87** [spacepy/spacepy#443]
My issue is when downloading/updating OMNI data, i.e. spacepy.toolbox.update(all=True) or  spacepy.toolbox.update(leapsecs=True), the process does not complete succesfully and a URLError is thrown.

Username of account is replaced for privacy/security.
 
### Minimal example to reproduce issue:

```
spacepy.toolbox.update(leapsecs=True)
```

### Error message/Traceback:

```
Traceback (most recent call last):

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 1350, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 1240, in request
    self._send_request(method, url, body, headers, encode_chunked)

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 1286, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 1235, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 1006, in _send_output
    self.send(msg)

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 946, in send
    self.connect()

  File "C:\Users\username\Anaconda3\lib\http\client.py", line 917, in connect
    self.sock = self._create_connection(

  File "C:\Users\username\Anaconda3\lib\socket.py", line 787, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):

  File "C:\Users\username\Anaconda3\lib\socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):

gaierror: [Errno 11001] getaddrinfo failed


During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "<ipython-input-10-ca0e0217ed14>", line 1, in <module>
    spacepy.toolbox.update(leapsecs=True)

  File "C:\Users\username\Anaconda3\lib\site-packages\spacepy\toolbox\__init__.py", line 907, in update
    u.urlretrieve(config['leapsec_url'], leapsec_fname)

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 247, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 542, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 502, in _call_chain
    result = func(*args)

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 1379, in http_open
    return self.do_open(http.client.HTTPConnection, req)

  File "C:\Users\username\Anaconda3\lib\urllib\request.py", line 1353, in do_open
    raise URLError(err)

URLError: <urlopen error [Errno 11001] getaddrinfo failed>
```

### OS, Python version, and dependency version information:

```
Windows-10-10.0.19041-SP0
sys.version_info(major=3, minor=8, micro=3, releaselevel='final', serial=0)
numpy=1.19.1
scipy=1.5.0
matplotlib=3.3.1
```

**Comment by Caseyb87** [comment]
Ah I see this is highly likely to be related to #435

**Comment by jtniehof** [comment]
Probably not #435 unless you've explicitly deleted your tai-utc file. This is because the USNO source for leap seconds has been down for over a year (was supposed to be back up by April last I checked); it's fixed in #364, which will be included in 0.2.2.

The tai-utc.dat included in spacepy 0.2.1 is up to date; we haven't had leap seconds since 2017 and the next earliest possible is June 2021. So you shouldn't need to update it.

**Comment by Caseyb87** [comment]
OK. Thanks!
